.github/workflows/check-license.yaml

```
@@ -20,12 +20,21 @@ jobs:
 
       - name: Check License Header
         run: |
-          CHECK=$(grep -rcL --exclude-dir={.git,build,**vernemq**} \
+          CHECK=""
+          for file in $(grep -rl --exclude-dir={.git,build,**vernemq**} \
             --exclude=\*.{crt,key,pem,zed,hcl,md,json,csv,mod,sum,tmpl,args} \
             --exclude={CODEOWNERS,LICENSE,MAINTAINERS} \
-            --regexp "Copyright (c) Abstract Machines" .)
+            .); do
+              
```

yamllint

[error] trailing spaces

(trailing-spaces)